README.md

```diff
@@ -2,8 +2,6 @@
 
 This repository contains all enzyme-constrained models generated by the @SysBioChalmers group.
 
-* Last update: 2018-10-12
-
 This repository is administered by Benjamín J. Sánchez ([@BenjaSanchez](https://github.com/benjasanchez)), Division of Systems and Synthetic Biology, Department of Biology and Biological Engineering, Chalmers University of Technology.
 
 ## Contributors
```

Comment:
Personally I prefer to just link to the [GitHub contributors](https://github.com/SysBioChalmers/ecModels/graphs/contributors) if possible.

Reply:
I agree, that's even much more informative that just indicating a data in the README.